src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -137,6 +137,7 @@ const createIdpButtons = (remediations) => {
     const button = {
       attributes: {
         'data-se': `social-auth-${type}-button`,
+        'title': displayName // We add a tooltip in case the name gets truncated if too long
```

Comment:
One problem with this is a screen reader will read both the button text **and** the tooltip `title`. If we can, we should try to avoid the duplicate reading.

@edburyenegren-okta - Do you have a suggestion here?

Reply:
Yeah great point! This didn't cross my mind. I'll wait for @edburyenegren-okta to chime in.

Reply:
This is partly the reason Odyssey uses a separate Tooltip component, but that feels far out of scope for this change.

One solution would be wrapping the truncated text leaf in a `span` with `aria-hidden`, and giving the `button` an `aria-label` or `title` (but not both) that contains the full, untruncated text. That would require swapping _all_ buttons over or detecting truncation somehow; both feel less than ideal...

Is Design comfortable with any other behaviors? Showing the full title on `:hover` or using a full Tooltip?